### PR TITLE
FOPTS-1577 AWS Unused RDS Instances: update policy name in the README

### PR DESCRIPTION
**Pull request by JesusParada**
### Description
Updated Policy Name in the README.md file from AWS Unused RDS Policy to AWS Unused RDS Instances

### Issues Resolved

Policy Name Updated

### Link to Example Applied Policy

Policy didn't applied, it's just a README update

### Contribution Check List

- [ ] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable
- [x] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: cost/aws/unused_rds/unused_rds.pt [hunk 1]
```diff
@@ -7,7 +7,7 @@ severity "low"
 category "Cost"
 default_frequency "weekly"
 info(
-  version: "6.1",
+  version: "6.2",
```
Comment (bryankaraffa) [inline cost/aws/unused_rds/unused_rds.pt:10]: We don't really need to bump the policy template version if it's only a README update fiw

File: cost/aws/unused_rds/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v6.2
+
+- Updated policy name in README from AWS Unused RDS Policy to AWS Unused RDS Instances
+
```
Comment (bryankaraffa) [inline cost/aws/unused_rds/CHANGELOG.md:6]:
```suggestion
```

File: cost/aws/unused_rds/unused_rds.pt (hunk 1 above)
Comment (bryankaraffa) [reply cost/aws/unused_rds/unused_rds.pt:10]:
```suggestion
  version: "6.1",
```

File: cost/aws/unused_rds/unused_rds_meta_parent.pt [hunk 3]
```diff
@@ -8,7 +8,7 @@ tenancy "single"
 default_frequency "15 minutes"
 info(
   provider: "AWS",
-  version: "6.1", # This version of the Meta Parent Policy Template should match the version of the Child Policy Template as it appears in the Catalog for best reliability
+  version: "6.2", # This version of the Meta Parent Policy Template should match the version of the Child Policy Template as it appears in the Catalog for best reliability
```
Comment (bryankaraffa) [inline cost/aws/unused_rds/unused_rds_meta_parent.pt:11]:
```suggestion
  version: "6.1", # This version of the Meta Parent Policy Template should match the version of the Child Policy Template as it appears in the Catalog for best reliability
```

File: cost/aws/unused_rds/README.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-# AWS Unused RDS Policy
+# AWS Unused RDS Instances
```
Comment (bryankaraffa) [inline cost/aws/unused_rds/README.md:1]: 👍

File: cost/aws/unused_rds/unused_rds.pt (hunk 1 above)
Comment (JesusParada) [reply cost/aws/unused_rds/unused_rds.pt:10]: Thanks for the info, I'll update it

File: cost/aws/unused_rds/README.md (hunk 4 above)
Comment (JesusParada) [reply cost/aws/unused_rds/README.md:1]: Done! I only updated the README file without any changelog and version update

**Pull request closed without merging**